package.json

```
@@ -1,13 +1,19 @@
 {
-  "name": "electron-gn-scripts",
-  "version": "0.0.0",
+  "name": "build-tools",
+  "version": "0.1.0",
   "main": "null",
-  "repository": "https://github.com/MarshallOfSound/electron-gn-scripts",
-  "author": "Samuel Attard <samuel.r.attard@gmail.com>",
+  "scripts": {
+    "lint": "for file in $(find nix -type f); do shellcheck --format=gcc $file; done;"
```

1. Would be better to invoke them in a batch -- it looks like shellcheck _may_ better handle some cases that way, e.g. false "unused variable" warnings for variables declared in one script and used by another.

```suggestion
    "lint": "find nix -type f | xargs shellcheck -f gcc -a"
```

2. Both of these variations require a sh-like environment and `find`, so I think `npm run lint` will fail on Windows? This isn't a blocker at this point, but IDK if there is a better way? :thinking: